docs/pages/versions/unversioned/sdk/notifications.mdx

```diff
@@ -149,7 +149,7 @@ async function registerForPushNotificationsAsync() {
       alert('Failed to get push token for push notification!');
       return;
     }
-    token = (await Notifications.getExpoPushTokenAsync()).data;
+    token = (await Notifications.getExpoPushTokenAsync({ projectId: 'your-project-id' })).data;
```

Comment:
```suggestion
    // Learn more about projectId:
    // https://docs.expo.dev/push-notifications/push-notifications-setup/#configure-projectid
    token = (await Notifications.getExpoPushTokenAsync({ projectId: 'your-project-id' })).data;
```

I think adding a link to the section where we explain projectId configuration is a good addition here.